containers/signing-service/install-dependencies.sh.j2

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+# Update package lists for Debian and install dependencies
+dependencies=(
+
+    # Packages needed to build python extensions
+    gcc
+    git
+    software-properties-common
```

Comment:
Please reorganize the scripts a bit so that we can remove `gcc`, `git`, `software-properties-common` and `python-dev` after the build:
- Instead of running `install-dependencies.sh` directly from `Dockerfile`, call it from `install-signing-service.sh`.
- Create a separate list for dependencies that can be removed after the build
- Create a script that removes dependencies after the build and run it from `install-signing-service.sh`
    - Move `apt-get clean`and similar to that script

Reply:
Done. The solution must be checked before merged when sign-service will complete on concent side.

Reply:
Please create a list of stuff that needs to be fixed before merging and put it in the pull request description. Otherwise it'll be easy to forget this stuff.